functions.opengraph.php

```diff
@@ -11,6 +11,10 @@
 add_action( 'wp_head', 'jetpack_og_tags' );
 
 function jetpack_og_tags() {
+	echo jetpack_og_get_tags();
+}
+
+function jetpack_og_get_tags() {
```

Comment:
yes, this was for unit testing. without it, I'd need to use output buffeting with e.g. `ob_start` to catch the `echo`ed output, which afaik is undesirable. I'm open to other ideas!